ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                        properties:
+                          ntpServer:
+                            description: ntpServer required by chrony manifest.
+                            type: string
+                      additionalNTPSources:
+                        description: |
+                          AdditionalNTPSources is a list of NTP sources (hostname or
+                          IP) to be added to all cluster hosts. They are added to any
+                          NTP sources that were configured through other means.
+                        type: array
+                        items:
+                          type: string
+                      ignitionConfigOverride:
+                        description: Json formatted string containing the user overrides for the initial ignition config.
+                        type: string
+                      nodes:
+                        description: List of the nodes that belong to the cluster.
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            hostName:
+                              description: hostName that will get applied to the node.
+                              type: string
+                            bmcAddress:
+                              description: |
+                                Address holds the URL for accessing the controller on the network. Must contain the redfish or the HW identification in the
+                                url for example redfish-virtualmedia+https://12.13.222.2/redfish/v1/Systems/System.Embedded.1
+                                example for Dell idrac-virtualmedia+https://10.16.231.87/redfish/v1/Systems/System.Embedded.1
+                              type: string
+                            bmcCredentialsName:
+                              description: Bmc credential secret info
+                              type: object
+                              properties:
+                                name:
+                                  description: "Name of the bmcCredentials secret"
+                                  type: string
+                            bootMACAddress:
+                              description: Which MAC address will PXE boot? This is optional for some types, but required for libvirt VMs driven by vbmc.
+                              type: string
+                              pattern: '[0-9a-fA-F]{2}(:[0-9a-fA-F]{2}){5}'
+                            bootMode:
+                              description: Select the method of initializing the hardware during boot. Defaults to UEFI.
+                              type: string
+                              default: UEFI
+                              enum:
+                                - UEFI
+                                - UEFISecureBoot
+                                - legacy
+                            cpuset:
```

Comment:
Eventually this will be specified per pool, not per node, but for SNO this is fine.

Reply:
will keep it per node for now